.github/workflows/package-component.yaml

```diff
@@ -62,8 +73,8 @@ jobs:
                 --registry-auth-password=${{ secrets.GITHUB_TOKEN }} \
                 --registry-url=ghcr.io \
                 --sdk-version=${{ steps.sdk_version.outputs.sdk_version }} \
-                --spec-path=new-components/${{ steps.tag_info.outputs.component_type }}/${{ steps.tag_info.outputs.component_name }}/component.yaml \
-                --version=${{ steps.tag_info.outputs.version }}
+                --version=${{ steps.tag_info.outputs.version }} \
```

Comment:
Perhaps we can also bake this into the components' binaries at build time and we can track this too like we do for the sdk-version?

Reply:
fixed